fastapi/params.py

```diff
@@ -50,8 +50,8 @@ def __init__(
             regex=regex,
             **extra,
         )
-
-
+    
```

Comment:
```suggestion

```

Reply:
https://softwareengineering.stackexchange.com/questions/121555/why-is-trailing-whitespace-a-big-deal#:~:text=While%20not%20strictly%20programming%2C%20whitespace,as%20input%20to%20something%20else.